server/schema.js

```diff
@@ -1,36 +1,79 @@
 const { gql } = require("apollo-server");
+const _ = require("lodash");
```

Comment:
Not that it matters here, but generally, we import only the things we use from `lodash`, so the rest of the package can be tree-shaked.

`const snakeCase = require('lodash/snakeCase');`